CONTRIBUTING.md

```diff
@@ -182,24 +170,13 @@ Here are the steps to release a beta version of this package depending on its ty
     - Meilisearch `prototype beta`: create a branch `prototype-beta/xx-xx`. Where `xxx` has the same name as the docker image containing the prototype.
         Example: If the [docker image](https://hub.docker.com/r/getmeili/meilisearch/tags) is named: `v0.29.0-pagination.beta.2`, the branch should be named: `prototype-beta/pagination`
 
-2. [Update the version](#version-update) following the correct format (X are numbers):
-    - package and prototype beta: `X.X.X-***.X`
-      example: `0.2.0-new-feature.0`
-    - pre-release beta: `X.X.X-vX.X.X-pre-release.X`
-      example: `0.2.0-v0.30.0-pre-release.0`
-
+2. Enable the pre-release mode by running `yarn changeset pre enter [X]`. `X` is the part after the `/` of your beta branch. Example for `beta/refactor`, X would be `refactor`. This will create a `pre.json` file in `.changesets` that must be pushed on your beta branch.
 
-3. Commit and push your code to the newly created branch (step 1).
+3. Commit and push your related PR's to the newly created branch (step 1).
 
-4. Go to the [GitHub interface for releasing](https://github.com/meilisearch/instant-meilisearch/releases): on this page, click on `Draft a new release`.
+4. When a PR is merged onto your beta branch, the [release CI](./.github/workflows/publish.yml) opens a PR named `Version Packages (X)` (see step 2 for `X`). This PR contains all the changesets and the version update based on the type of changes in the changesets.
 
-5. Create a GitHub pre-release:
-  - Fill the description with the detailed changelogs
-  - Fill the title with the version defined on step `2`, appened with a `v`. Ex: `v0.1.0`
-  - Fill the tag with the same name as the title
-  - ⚠️ Select the branch created on step `1` and NOT `main`
-  - ⚠️ Click on the "This is a pre-release" checkbox
-  - Click on "Publish release"
+5. To publish the release on Github and `npm`, you need to merge the `Version Packages (X)` PR. This will trigger the publishing.
```

Comment:
So if I understand correctly
- for each PR merged, there is another created PR
- once the second PR is merged, a release is done automatically -> no manual release here?

Reply:
For every PR merged with a changeset, the PR named `Version Packages` is updated automatically with the new changesets. 
Once you want to release, you merge the `Version Packages` PR.